calcom/cal.com
# fix: #11586 updated UX/UI of filters on the bookings page

Updated filter UI/UX to match Figma.

## What does this PR do?

Fixes #11586 

Fix UI/UX of filters on the bookings page, following figma file to implement the desired design.

[Loom Video](https://www.loom.com/share/e2ad57c864a54b48b047c47fb72bc748?sid=ff6e6115-127f-476d-b988-7abaaba1e001)

## Requirement/Documentation



[Figma File](https://www.figma.com/file/UajFu4M1APhn2AywAEJkJr/New-Features?type=design&node-id=7869%3A93929&mode=dev)

## Type of change


## How should this be tested?



Tested switching between pages and status to verify the state of the filters is reseted.
Changed the page size to verify the buttons doesn't jump around.
Selected filters to verify the count of the filters is correct.

Tested building the project and it everything was ok.

## Mandatory Tasks


## Checklist



- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
